### ISSUE TITLE
Open the appropriate settings from the Quick Toolbar

### ISSUE THREAD

**Issue by nk-tedo-001** [ProxymanApp/Proxyman#2041]
Option or CMD click should open the tool's window if it has one.

For example - simple click on Breakpoint to activate it and Option/CMD click to open Breakpoint Rules.

**Comment by NghiaTranUIT** [comment]
@nk-tedo-001 not sure why you don't use built-in shortcut to open the Breakpoint Windows by using Command+Option+B ?

**Comment by nk-tedo-001** [comment]
> [@nk-tedo-001](https://github.com/nk-tedo-001) not sure why you don't use built-in shortcut to open the Breakpoint Windows by using Command+Option+B ?

Yes, I could use it, but I would have to remember each shortcut, even the ones I barely use.
Instead of a simple cmd-click to open Network Condition options.
